portfolio.css

```diff
@@ -39,6 +39,75 @@ body {
   .name {
     width: 327px;
   }
+
+  .work-section .work-header {
+    width: 327px;
+    line-height: 75px;
+  }
+
+  .bottom-border {
+    margin: 10px;
+    opacity: 1;
+  }
+
+  .work-section {
+    display: flex;
```

Comment:
I would suggest you use `display : grid` to arrange your cards. For this project, it is a mandatory requirement for you to pass. See the highlighted bit of the screenshot of the requirements below:
<img width="1210" alt="Screenshot 2022-10-01 at 18 09 43" src="https://user-images.githubusercontent.com/48546992/193416017-510ac745-4795-4dd5-bfde-96545189fba3.png">